### PR TITLE
AccountController refactoring.

### PR DESCRIPTION
**Pull request by Teodor92**
- Removed old unused code.
- Removed Recaptcha package.
- Extracted constants and resources.

### CODE REVIEW

File: Open Judge System/Web/OJS.Web/Controllers/AccountController.cs [hunk 1]
```diff
@@ -92,91 +81,30 @@ public ActionResult Login(string returnUrl)
...
-        // GET: /Account/Manage
+        [HttpGet]
         public ActionResult Manage()
         {
-            return this.RedirectToAction("ExternalNotify", "Account", new { area = string.Empty });
-
-            this.ViewBag.HasLocalPassword = this.HasPassword();
-            this.ViewBag.ReturnUrl = Url.Action("Manage");
-            return this.View();
+            return this.RedirectToExternalSystemMessage();
```
Comment (KristianMariyanov) [inline Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: Expression body to all one line actions

File: Open Judge System/Web/OJS.Web/Controllers/AccountController.cs [hunk 2]
```diff
@@ -635,6 +243,11 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        private ActionResult RedirectToExternalSystemMessage()
+        {
+            return this.RedirectToAction("ExternalNotify", "Account", new { area = string.Empty });
```
Comment (KristianMariyanov) [inline Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: nameof

**Comment by Teodor92** [comment]
Done.

File: Open Judge System/Web/OJS.Web/Controllers/AccountController.cs [hunk 3]
```diff
@@ -45,30 +35,29 @@ public AccountController(IOjsData data, UserManager<UserProfile> userManager)
 
         private IAuthenticationManager AuthenticationManager => this.HttpContext.GetOwinContext().Authentication;
 
-        // GET: /Account/Login
+        [HttpGet]
```
Comment (KristianMariyanov) [inline Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: @jicata do you add `[HttpGet]` on get actions here?

Comment (jicata) [reply Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: It seems to be a somewhat weakly followed convention (_some_ `GET` actions in _some_ controllers have the annotation, most don't). I personally don't see a reason to enforce it. What  do you think?

Comment (Teodor92) [reply Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: In my experience, stating something explicitly is better than relying on implicitly inferring it from context, as it reduces cognitive strain on the reader.

Comment (KristianMariyanov) [reply Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: We skip `[HttpGet]` in Suls for sure, but I'm not sure do you have convention for this in Judge

Comment (jicata) [reply Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: Even if that convention had existed, I'm inclined to forgo and abandon it, in the light that we are aiming at unifying conventions across SULS and OJS. As a result, @Teodor92 you can safely remove the annotations.

**Pull request merged**